neo/Network/RPC/RpcServer.cs

```diff
@@ -443,12 +442,20 @@ private JObject ProcessRequest(HttpContext context, JObject request)
             return response;
         }
 
-        public void Start(IPAddress bindAddress, int port, string sslCert = null, string password = null, string[] trustedAuthorities = null)
+        public void Start(IPAddress bindAddress, int port, string sslCert = null, string password = null, string[] trustedAuthorities = null, int maxConcurrentConnections = default)
         {
             host = new WebHostBuilder().UseKestrel(options => options.Listen(bindAddress, port, listenOptions =>
             {
                 // Default value is unlimited
-                options.Limits.MaxConcurrentConnections = MaxConcurrentConnections;
+                if (maxConcurrentConnections == 0)
+                {
+                    options.Limits.MaxConcurrentConnections = Peer.DefaultMaxConnections;
```

Comment:
I feel that we are mixing RPC with P2P, I prefer to set the value.

Reply:
> I feel that we are mixing RPC with P2P, I prefer to set the value.

Agree.